### PR TITLE
Igor static.analysis

### PR DESCRIPTION
**Pull request by IgorGanapolsky**
Reformat code.

### CODE REVIEW

File: mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java [hunk 1]
```diff
@@ -310,7 +314,7 @@ public void start(int port) throws IOException {
    * @param port the port to listen to, or 0 for any available port. Automated tests should always
    * use port 0 to avoid flakiness when a specific port is unavailable.
    */
-  public void start(InetAddress inetAddress, int port) throws IOException {
+  private void start(InetAddress inetAddress, int port) throws IOException {
```
Comment (artem-zinnatullin) [inline mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java]: This is public API

Comment (IgorGanapolsky) [inline mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java]: The InetAddress param method is exposed to public?

**Comment by swankjesse** [comment]
Not sure what the goal is here.

**Comment by IgorGanapolsky** [comment]
@swankjesse You don't believe in static analysis?

**Comment by swankjesse** [comment]
I’m sorry, your PR appears to just reformat the code. I don't believe in reformatting the code.

**Comment by IgorGanapolsky** [comment]
There were a couple of methods that were throwing exceptions for no
reason.  And a couple of generic types that do not need to be there.  And
imports that weren't organized into proper blocks.

**Comment by JakeWharton** [comment]
Imports should not be put into blocks.

As for the other things, submit individual PRs which change only the
problematic section.

**Pull request closed without merging**